docs/src/reference/python/operations/logic/allclose_example.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import equistore, numpy as np
+from equistore import TensorBlock, TensorMap, Labels, sum_over_samples
+from equistore import allclose, allclose_raise, allclose_block, allclose_block_raise
+
+# Create some TensorBlocks
+
+# Create simple block1
+block1 = TensorBlock(
+    values=np.array([
+         [1, 2, 4],
+         [3, 5, 6],
+     ]),
+    samples=Labels(
+         ["structure", "center"],
+         np.array([
+             [0, 0],
+             [0, 1],
+         ]),
+     ),
+    components=[],
+    properties=Labels(
+        ["properties"], np.array([[0], [1], [2]])
+     ),
+)
+keys = Labels(names=["key"], values=np.array([[0]]))
+
+# Recreate block1, but rename properties label 'properties' to 'PROPERTIES'
+block2 = TensorBlock(
+    values=np.array([
+         [1, 2, 4],
+         [3, 5, 6],
+     ]),
+    samples=Labels(
+         ["structure", "center"],
+         np.array([
+             [0, 0],
+             [0, 1],
+         ]),
+     ),
+    components=[],
+    properties=Labels(
+        ["PROPERTIES"], np.array([[0], [1], [2]])
+     ),
+)
+
+allclose_block(block1, block2)
+#Output:
+# False
+### This fails because allclose_block checks that the properties of two blocks are equal too
+
+# Using allclose_block_raise raises a ValueError when the check fails, and provides additional information
+# as to why it fails
+allclose_block_raise(block1, block2)
+#Output:
+# ValueError: Inputs to 'allclose' should have the same properties:
+# properties names are not the same or not in the same order.
+
+# Recreate block1, but change first value in the block from 1 to 1.00001
+block3 = TensorBlock(
+    values=np.array([
+         [1.00001, 2, 4],
```

Comment:
Maybe it visually easier to grasp your examples below when you define 1.00001 as 1+1e-5

Reply:
Très bon idée